modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -2,67 +2,54 @@
+
+To obtain custom event notifications:
 
-. Create a streaming channel with the Publish Streaming Channel operation.
+. Create a streaming channel by using the `Publish Streaming Channel` operation.
 +
 A StreamingChannel is a special Salesforce object that represents a channel used for notifying listeners of generic Streaming API events.
 +
-Note that you can also create a streaming channel through the Salesforce or through Workbench.
+You can also create a streaming channel through Salesforce or https://workbench.developerforce.com/about.php[
+workbench].
 +
-. Subscribe to the channel through the `Subscribe Channel Listener` operation.
+. Subscribe to the channel by using the `Subscribe Channel Listener` operation.
 +
-The Salesforce connector converts the custom events in your streaming channel to Mule events and dispatches them to your flows.
+Salesforce Connector converts the custom events in your streaming channel to Mule events and sends them to your flows.
+
+For more information about working with streaming channels, see <<streamingchanel,Create a Streaming Channel to Receive Data From Salesforce>> and <<receivedata,Receive Data From Salesforce>>.
 
 [[eventsandtopics]]
 == Events and Topics
 
-Your application can receive events by subscribing to a Salesforce topic.
-
-Each event that travels through your flows contains information about the Salesforce data that has changed, how it changes, and when. The connector parses this information and sends you information that a flow can work with.
-
-Inbound properties of events:
+Your app can receive events by subscribing to an existing Salesforce topic or by creating a new topic and then subscribing to it.
 
-* `payload`
-* `createdDate`
-* `replayId`
+Each event that travels through your flow contains information about the Salesforce data that changed, including how the data changed and when the change occurred. The connector parses this information and sends your app the information.
 
-Salesforce stores events for 24 hours, so you can retrieve stored events during that retention window. A subscriber (to a topic or channel) can retrieve events at any time and is not restricted to listening to events at the time they are sent.
+Salesforce stores events for 24 hours. You can retrieve stored events during that retention window. A subscriber to a topic or channel can retrieve events at any time and is not restricted to listening to events at the time they are sent.
 
-Each broadcast event is assigned a numeric ID. IDs are incremented and not guaranteed to be contiguous for consecutive events. Each ID is guaranteed to be higher than the ID of the previous event. For example, the event following the event with ID 999 can have an ID of 1025. The ID is unique for the organization and the channel. The IDs of deleted events are not reused.
+*Reviewers: the above paragraph is not clear. Does the subscriber have to retrieve events within 24 hours?*
```

Comment:
The idea is that a user can create a topic for accounts, for example. He does not subscribe to that topic immediately. He creates or updates accounts and these events are pushed on the topic he created - these events live on the topic for 24 hours, if he subscribes during these 24 hours he will be able to receive all the events. If the 24 hours period passes he won't be able to receive those events anymore but he will receive newer events(that are not yet expired = that occurred in the past 24 hours in relation with the subscription time)